### ISSUE TITLE
Online Grocery Recommendation

### ISSUE THREAD

**Issue by pratishthaa** [Niketkumardheeryan/ML-CaPsule#284]
Based on the similarity of what people buy, recommend grocery.

**Comment by Snossy123** [comment]
I am a participant in GSSo'22 
I want to work on this issue please assign it to me
I can solve it with association role technique.

**Comment by Neha-Niharika-Kar** [comment]
Is this project updated? I can't find this on the repository. Can you please help me?
What update is needed to be done now?

**Comment by pratishthaa** [comment]
You have to work on this issue yourself. It's not added n the repository.

**Comment by Neha-Niharika-Kar** [comment]
Okay thanks, what is the deadline to complete?

**Comment by pratishthaa** [comment]
6 days..keep updating in between!

**Comment by Neha-Niharika-Kar** [comment]
Is there any specific dataset to use?

**Comment by Rutikab12** [comment]
> Is there any specific dataset to use? ipynb file upload?

Yes file should be ipynb  and if you have any related dataset for the project, you can use that.

**Comment by Neha-Niharika-Kar** [comment]
Okay thank you

**Comment by A-b-i-r-a-m-i-G-S** [comment]
@pratishthaa I would like to work on this problem statement. Could you please assign the issue to me as part of gssoc'24?
